domains/new_dev_project/application/data_insertion_controller.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.append('../../..')
+
+from domains.new_dev_project.core import db_insertion_facade
+from utils.common_utils import get_request_id
+import utils.logger as logger
+
+def dump_employee_increase_per_quater_per_domain(event):
```

Comment:
dump is not a valid name for a function. it can be create or upsert or persist etc.